practices/java-web/HelloWorldServletMvcWebApplication/src/main/java/com/dowa/java/web/connection/CreateConnection.java

```diff
@@ -0,0 +1,40 @@
+package com.dowa.java.web.connection;
+
+/**
+ * Created by Tonoid on 5/20/2015.
+ */
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+
+public class CreateConnection {
```

Comment:
Is this class used or are you using ConnectionFactory instead? If this is not used, you can remove.